### DISCUSSION TITLE
how do i install the latest version of xrdp on linux mint cinnamon

### DISCUSSION THREAD

**Discussion by Aerospace911** [neutrinolabs/xrdp#2749] [Q&A] [answered]
title

**Comment by matt335672** [comment, marked as answer]
We don't provide packages for an operating systems as part of this project, and as a project we're not in a position to recommend any community builds.

You can build it yourself from source [here](https://github.com/neutrinolabs/xrdp/wiki/Building-on-Debian-8), possibly dated.
